### ISSUE TITLE
My watch has stopped vibrating

### ISSUE THREAD

**Issue by Mararjona** [Rytiggy/Glance#87]
Hello, my versa Lite wacht has stopped vibrating in all alerts with no reason. I have checked the configuration and everything is fine. I don,t know what,s happen.

**Comment by Rytiggy** [comment]
Is your phone in do not disturb mode or silent mode?

**Comment by Mararjona** [comment]
No, my phone is working fine. Tomato alarms are ringing correctly in my phone but no vibe in my wacht
